definitions/linux.yaml

```diff
@@ -137,6 +137,116 @@ labels: [Authentication, Configuration Files]
 provides: [users.username]
 supported_os: [Linux]
 ---
+name: ProcMounts
+doc: Current mounted filesystems.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/mounts'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysFsSecurity
+doc: Filesystem hardening settings.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/fs/protected_hardlinks'
+      - '/proc/sys/fs/protected_symlinks'
+      - '/proc/sys/fs/suid_dumpable'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysKernelBootloader
+doc: Bootloader state acquired from the kernel.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/bootloader_type'
+      - '/proc/sys/kernel/bootloader_version'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysKernelMemoryPrediction
+doc: Memory address obfuscation settings.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/dmesg_restrict'
+      - '/proc/sys/kernel/kptr_restrict'
+      - '/proc/sys/kernel/randomize_va_space'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysKernelModules
+doc: Module loading options and taint state.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/kexec_load_disabled'
```

Comment:
Same goes for all the rest of these.  They should only be one artifact if they are very tightly related like: protected_hardlinks and symlinks; and
bootloader_type and version.